README.md

```diff
@@ -137,8 +141,8 @@ should be used:
     | AMI           | Amazon Region  |
     | ---           | ---            |
     | ami-2051294a* | us-east-1*     |
-    | ami-d1315fb1  | us-west-1      |
-    | ami-775e4f16  | us-west-2      |
+    | ami-d1315fb1  | us-west-1      | N. California
+    | ami-775e4f16  | us-west-2      | Oregon 
```

Comment:
So here's another problem.... you say us-west-1 but in the AMS console, they don't call it that.   So I worked backwards from this instructions and found out what is the name.   I don't know what they call it in the east or other regions because I didn't try to make it work there.